### PR TITLE
mbedTLS: add support for CURLOPT_CAINFO_BLOB

### PR DESCRIPTION
**Pull request by Floni**

### CODE REVIEW

**Comment by Floni** [comment]
Not sure about the failing tests, they seem unrelated to the PR. Could they be caused by server hiccups?

**Comment by bagder** [comment]
@Floni Right, those are (unfortunately) just our regular flaky builds and they're not failing because of your code.

File: docs/libcurl/opts/CURLOPT_CAINFO_BLOB.3 [hunk 1]
```diff
@@ -62,7 +62,7 @@ if(curl) {
 Added in 7.77.0.
 
 This option is supported by the BearSSL (since 7.79.0),
-OpenSSL, Secure Transport and Schannel backends.
+mbedTLS, OpenSSL, Secure Transport and Schannel backends.
```
Comment (bagder) [inline docs/libcurl/opts/CURLOPT_CAINFO_BLOB.3:65]: maybe add `(since 7.81.0)` there for mbedTLS?

**Review by bagder — APPROVED** [review]

**Comment by bagder** [comment]
Thanks!

**Comment by bagder** [comment]
Did you try test 678 with this code applied? It seems it fails now: https://curl.zuul.vexxhost.dev/build/a9732e2c883a4672aabd1c89dc4e40bd

The exact error output is:
~~~
 * Error importing ca cert blob  - mbedTLS: (-0x2180) X509 - The CRT/CRL/CSR format is invalid, e.g. different type expected

**Comment by bagder** [comment]
create an issue

**Pull request closed without merging**